backend/src/schema/types/type/User.gql

```diff
@@ -157,6 +159,7 @@ type Query {
 }
 
 type Mutation {
+  acceptTermsAndCondition(version: String!): Boolean
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-16T08:39:38Z" title="Friday, August 16th 2019, 10:39:38 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -157,6 +159,7 @@ type Query {
 }
 
 type Mutation {
+  acceptTermsAndCondition(version: String!): Boolean
```


```suggestion
```
We can use `UpdateUser` for this because we're changing properties on the user object.